### ISSUE TITLE
Error when shutting down Kernel (v0.1.10)

### ISSUE THREAD

**Issue by JimVanEeden** [aionnetwork/aion#17]
I'm getting this error when shutting down the kernel:

```
18-02-13 16:58:30.665 INFO  GEN  [Shutdown]: Starting shutdown process...
18-02-13 16:58:30.666 INFO  GEN  [Shutdown]: Shutting down zmq ProtocolProcessor
18-02-13 16:58:33.666 INFO  GEN  [Shutdown]: Shutdown zmq ProtocolProcessor... Done!
18-02-13 16:58:33.666 INFO  GEN  [Shutdown]: Shutting down sealer
18-02-13 16:58:33.666 INFO  CONS [Shutdown]: sealer stopping 🔒
18-02-13 16:58:33.666 INFO  CONS [Shutdown]: Interrupt sealer 1
18-02-13 16:58:33.666 INFO  CONS [Shutdown]: Interrupt sealer 2
18-02-13 16:58:33.666 INFO  CONS [Shutdown]: Interrupt sealer 3
18-02-13 16:58:33.666 INFO  CONS [Shutdown]: Interrupt sealer 4
18-02-13 16:58:33.666 INFO  CONS [Shutdown]: Interrupt sealer 5
18-02-13 16:58:33.666 INFO  CONS [Shutdown]: Interrupt sealer 6
18-02-13 16:58:34.855 INFO  CONS [Shutdown]: Stopped sealer 1
18-02-13 16:58:34.855 INFO  CONS [Shutdown]: Stopped sealer 2
18-02-13 16:58:35.447 INFO  CONS [Shutdown]: Stopped sealer 3
18-02-13 16:58:35.447 INFO  CONS [Shutdown]: Stopped sealer 4
18-02-13 16:58:35.488 INFO  CONS [Shutdown]: Stopped sealer 5
18-02-13 16:58:35.488 INFO  CONS [Shutdown]: Stopped sealer 6
18-02-13 16:58:35.488 INFO  GEN  [Shutdown]: Shutdown sealer... Done!
18-02-13 16:58:35.488 INFO  GEN  [Shutdown]: Shutting down the AionHub...
18-02-13 16:58:35.488 INFO  GEN  [Shutdown]: <KERNEL SHUTDOWN SEQUENCE>
18-02-13 16:58:35.488 INFO  GEN  [Shutdown]: <SYNC> shutting down syncMgr...
18-02-13 16:58:35.488 INFO  GEN  [Shutdown]: <SYNC> shutdown syncMgr... Done!
18-02-13 16:58:35.488 INFO  GEN  [Shutdown]: TransactionExecThread shutting down...
18-02-13 16:58:35.489 INFO  GEN  [Shutdown]: TransactionExecThread waiting termination.
18-02-13 16:58:35.489 INFO  GEN  [Shutdown]: TransactionExecThread shutdown... Finished!
18-02-13 16:58:35.489 INFO  GEN  [Shutdown]: <shutdown-tx>
....18-02-13 16:58:39.490 INFO  GEN  [Shutdown]: shutting down consensus...
18-02-13 16:58:39.490 INFO  GEN  [Shutdown]: shutdown consensus... Done!
18-02-13 16:58:39.490 INFO  GEN  [Shutdown]: shutting down DB...
18-02-13 16:58:39.491 INFO  GEN  [Shutdown]: details DB closed.
18-02-13 16:58:39.546 INFO  GEN  [Shutdown]: state DB closed.
18-02-13 16:58:39.546 INFO  GEN  [Shutdown]: transaction DB closed.
Exception in thread "pow" java.lang.RuntimeException: Database is not opened: LevelDBWithCache:<autocommit=OFF,size=UNBOUND,stats=OFF>
	at org.aion.dbmgr.driver.AbstractDatabaseWithCache.check(AbstractDatabaseWithCache.java:135)
	at org.aion.dbmgr.driver.AbstractDatabaseWithCache.get(AbstractDatabaseWithCache.java:466)
	at org.aion.dbmgr.driver.AbstractDatabaseWithCache.get(AbstractDatabaseWithCache.java:25)
	at org.aion.trie.JournalPruneDataSource.get(JournalPruneDataSource.java:218)
	at org.aion.trie.JournalPruneDataSource.get(JournalPruneDataSource.java:40)
	at org.aion.trie.Cache.get(Cache.java:99)
	at org.aion.trie.TrieImpl.getNode(TrieImpl.java:465)
	at org.aion.trie.TrieImpl.get(TrieImpl.java:240)
	at org.aion.trie.TrieImpl.get(TrieImpl.java:172)
	at org.aion.trie.SecureTrie.get(SecureTrie.java:41)
	at org.aion.db.a0.AionRepositoryImpl.getAccountState(AionRepositoryImpl.java:454)
	at org.aion.db.a0.AionRepositoryImpl.loadAccountState(AionRepositoryImpl.java:479)
	at org.aion.db.AbstractRepositoryCache.getAccountState(AbstractRepositoryCache.java:97)
	at org.aion.db.AbstractRepositoryCache.addBalance(AbstractRepositoryCache.java:202)
	at org.aion.a0.AionBlockchainImpl.addReward(AionBlockchainImpl.java:899)
	at org.aion.a0.AionBlockchainImpl.generatePreBlock(AionBlockchainImpl.java:849)
	at org.aion.a0.AionBlockchainImpl.createNewBlock(AionBlockchainImpl.java:503)
	at org.aion.pow.AionPoW.createNewBlockTemplate(AionPoW.java:212)
	at org.aion.pow.AionPoW.lambda$init$0(AionPoW.java:86)
	at java.base/java.lang.Thread.run(Thread.java:844)
18-02-13 16:58:39.558 INFO  GEN  [Shutdown]: block DB closed.
18-02-13 16:58:39.558 INFO  GEN  [Shutdown]: shutdown DB... Done!
```

Not sure what it is, but thought I'd post an issue for it.

**Comment by AlexandraRoatis** [comment]
Thank you for reporting the issue. It is a known bug that will be fixed in the next release.

**Comment by AlexandraRoatis** [comment]
Bug fixed in v0.1.11
